iis-wf/iis-wf-ingest-pmc/src/test/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/MetadataImporterMain.java

```diff
@@ -0,0 +1,84 @@
+package eu.dnetlib.iis.wf.ingest.pmc.metadata;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import org.apache.avro.io.DatumWriter;
+import org.apache.avro.io.Encoder;
+import org.apache.avro.io.EncoderFactory;
+import org.apache.avro.specific.SpecificDatumWriter;
+import org.apache.commons.lang.StringUtils;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+import org.xml.sax.XMLReader;
+
+import eu.dnetlib.iis.ingest.pmc.metadata.schemas.ExtractedDocumentMetadata;
+
+/**
+ * This class is responsible for performing metadata extraction out of the JATS XML file which location is provided as input parameter.
+ * @author mhorst
+ */
+public class MetadataImporterMain {
+    
+    private static final String encoding = "utf-8"; 
+    
+    public static void main(String[] args) throws IOException, ParserConfigurationException, SAXException {
+        if (args.length >= 1) {
```

Comment:
Since it only uses one argument it should check that there is exactly one. It will also help to catch forgotten quoting - although it's unlikely to have a path containing whitespace.